routes/cartRouter.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const router = express.Router();
+const { verifyJWT } = require("../middleware/auth");
+const { catchAsync } = require("../middleware/error");
+
+const cartController = require("../controllers/cartController");
+router.get("", verifyJWT, catchAsync(cartController.getCartList));
+router.patch(
+  "/:cartId",
+  verifyJWT,
+  catchAsync(cartController.updateCartQuantity)
+);
+router.delete("/:cartId", verifyJWT, catchAsync(cartController.deleteCartList));
+router.patch("", catchAsync(cartController.deleteCartItems));
```

Comment:
```suggestion
const express = require("express");

const { verifyJWT } = require("../middleware/auth");
const cartController = require("../controllers/cartController");

const router = express.Router();

router.get("", verifyJWT, cartController.getCartList);
router.patch(
  "/:cartId",
  verifyJWT,
  cartController.updateCartQuantity
);
router.delete("/:cartId", verifyJWT, cartController.deleteCartList);
router.patch("", cartController.deleteCartItems);
```
1. 줄바꿈 및 모듈 import이나 require 순서 컨벤션에 맞춰 통일되게 작성하면 보다 가독성 좋은 코드가 됩니다!
2. catchAsync와 같은 모듈은 router 내에 위치하기보다 controller 모듈 file에서 작성될 수 있게 수정해주세요! 동작에는 지금처럼 작성되어도 문제 없을 수 있습니다. 다만, 실제 프로그래밍을 하는데 있어서 지금은 class를 적용하지 않고 함수로서 구현을 하고 있지만 단일책임원칙이라는 개념이 있습니다. 이는 하나의 클래스 혹은 객체는 하나의 책임(역할)만 지녀야한다를 의미하는데, 지금의 코드에 대입하여 간단히 설명드리자면, 가급적 router라는 모듈은 단지 routing을 하는 역할만 가지고 있도록 지향해야 한다는 의미입니다. 따라서 catchAsync 모듈은 router보다는 controller 모듈에서 직접작성될 수 있도록 수정해주세요 :)
3. deleteCartList, deleteCartItems 두가지가 있는데 각각 어떤기능인지 알 수 있을까요? 만약 각각 전체삭제와 선택삭제를 의미하는거라면 하나의 모듈로 합칠 수 있습니다.

Reply:
3번의 경우 deleteCartList는 단일 아이템 ID만 삭제 하려고 넣었습니다.
deletecartItems의 경우 선택 삭제가 되게끔 요청을 받아서 추가해서 넣었습니다!

하나로 합칠 경우에는 deleteCartList를 지우는게 좋을까요?

Reply:
const { cartId } = req.query
// DELETE localhost:3000/carts?cartId=1&cartId=2&cartId=3&&cartId=5
// DELETE localhost:3000/carts?cartId=2

cartId = 2
(1, 3, 5)